spec/features/tool_drafts/forms/tool_draft_form_navigation_spec.rb

```diff
@@ -0,0 +1,251 @@
+include DraftsHelper
+
+describe 'Tool Draft form navigation', js: true do
+  form_titles = ['Tool Information', 'Related URLs', 'Compatibility and Usability', 'Descriptive Keywords', 'Tool Organizations', 'Tool Contacts', 'Smart Handoff Information']
+  form_names = %w[tool_information related_urls compatibility_and_usability descriptive_keywords tool_organizations tool_contacts smart_handoff_information]
+
+  context 'when visiting the edit page for a full tool draft' do
+    let(:full_draft) { create(:full_tool_draft) }
```

Comment:
Had to look up the differences between let and instance variables. We should be using let more, I think.

Reply:
i agree